packages/appeal-reply-service-api/src/lib/notify.js

```diff
@@ -0,0 +1,56 @@
+const NotifyBuilder = require('@pins/common/src/lib/notify/notify-builder');
+const axios = require('axios');
+const config = require('./config');
+const logger = require('./logger');
+
+async function sendAppealReplySubmissionConfirmationEmailToLpa(reply) {
+  let appeal = {};
+  let lpa = {};
+  let pdf = '';
+
+  try {
+    const appealUrl = `${config.appeals.url}/api/v1/appeals/${reply.appealId}`;
+    const appealRes = await axios.get(appealUrl);
+    appeal = appealRes.data;
+
+    const lpaUrl = `${config.appeals.url}/api/v1/local-planning-authorities/${appeal.lpaCode}`;
+    const lpaRes = await axios.get(lpaUrl);
+    lpa = lpaRes.data;
+
+    const pdfUrl = `${config.docs.api.url}/api/v1/${reply.id}/${reply.submission.pdfStatement.uploadedFile.id}/file`;
+    logger.info({ docsPath: pdfUrl }, 'docs service route');
+
+    const pdfRes = await axios.get(pdfUrl, { responseType: 'blob' });
+    pdf = pdfRes.data;
+  } catch (err) {
+    logger.error({ err }, 'Unable to get data required to send email.');
```

Comment:
At a later date it might be useful to get some seperation of why the error occurred, i.e. which service / piece of data.

Reply:
This is now separated as suggested :+1: